src/main/resources/driven-adapter/dynamo-db/build.gradle.mustache

```diff
@@ -1,7 +1,7 @@
 dependencies {
     implementation project(':model')
     implementation 'org.springframework:spring-context'
-    implementation 'com.amazonaws:aws-java-sdk-dynamodb:1.12.145'
+    implementation 'software.amazon.awssdk:dynamodb'
```

Comment:
Thanks for comments, I added software.amazon.awssdk:dynamodb-enhanced library, next, I modified build.gradle.mustache and finally I deleted/reversed change in build.gradle.kts.mustache file

Reply:
@dhfherna when you are changing or creating a module you should test it in you local machine, because there you can see the real behaviour of your changes and your generated files, we apologize not having a good documentation related to it before, but we are happy of the new [guide](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Implementing-or-changing-a-module) in our [contributing page](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing), we spect that it will be of help to you.